pilot/cmd/pilot-discovery/main.go

```diff
@@ -130,14 +130,6 @@ func init() {
 		"Discovery service grpc address, with https")
 	discoveryCmd.PersistentFlags().StringVar(&serverArgs.DiscoveryOptions.MonitoringAddr, "monitoringAddr", ":9093",
 		"HTTP address to use for the exposing pilot self-monitoring information")
-	discoveryCmd.PersistentFlags().BoolVar(&serverArgs.DiscoveryOptions.EnableProfiling, "profile", true,
```

Comment:
What happened with this flag ? We should still be able to turn off pprof and debug interfaces.

Reply:
It was only used in v1 discovery. But if we want to keep the flag around for future use, that makes sense.